### PR TITLE
Feature/one hot encoder

### PR DESCRIPTION
**Pull request by JaeSooLEE**
ajout du one hot encoder transformer et son unittest

### CODE REVIEW

File: preprocessing/transformers/onehot_encoder_transformer.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    def fit(self, df=None, y=None):
+        return self
+
+    def transform(self, df):
+        try:
+            result = pd.DataFrame()
+            dfc = df.copy()
+            for feature_name in self.columns_to_encode:
+                result = pd.concat([result, pd.get_dummies(dfc[feature_name], feature_name)], axis=1)
+            print(result)
```
Comment (Alain-TC) [inline preprocessing/transformers/onehot_encoder_transformer.py]: To be removed

File: preprocessing/transformers/onehot_encoder_transformer.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import pandas as pd
+import numpy as np
+from sklearn.base import TransformerMixin
+
+
+class OneHotTransformer(TransformerMixin):
+    def __init__(self, columns_to_encode):
+        self.columns_to_encode = columns_to_encode
+
+    def fit(self, df=None, y=None):
```
Comment (Alain-TC) [inline preprocessing/transformers/onehot_encoder_transformer.py:29]: Comme pour Normalize, il faut récupérer les infos dans le fit avant, pour des raisons similaires (on en discute si besoin)

**Review by Alain-TC — CHANGES_REQUESTED** [review]

File: main.py [hunk 3]
```diff
@@ -38,18 +39,21 @@
                             "WoodDeckSF","OpenPorchSF","EnclosedPorch","3SsnPorch","ScreenPorch","PoolArea","MiscVal",
                            "MoSold","YrSold","LotArea"]
 
-    processing_pipeline = make_pipeline(KeepColumnsTransformer(quantitative_columns),
+    qualitative_columns = df_train.columns.drop(quantitative_columns).drop(['SalePrice'])
```
Comment (Alain-TC) [inline main.py]: A récupérer explicitement. (traitons variables quali et quanti au même titre)

**Review by Alain-TC — CHANGES_REQUESTED** [review]

**Review by Alain-TC — APPROVED** [review]
